### ISSUE TITLE
Customizable action where the field is placed

### ISSUE THREAD

**Issue by Saarnaki** [ouun/wc-repack#1]
Hi

Appreciate the idea behind this plugin.

Most of the filters and customization is good. There is 1 thing I find could be improved - ability to specify where the checkbox (_add_checkout_repack_field_ method) is being called.

I'd like to switch it from **woocommerce_after_order_notes** to **woocommerce_review_order_before_submit** in my case.

**Comment by ouun** [comment]
Thanks @Saarnaki, that is a useful suggestion.
I’ll have a look at this ASAP. But you are also very welcome to open a PR with the required changes.

**Comment by ouun** [comment]
Hi @Saarnaki!
I just released version 1.0.4 (891db34) what allows you to use the `repack_checkout_consent_position` filter, an example below. Can you please test and give some feedback.

Thanks!

```
add_filter( 'repack_checkout_consent_position', function () {
    return 'woocommerce_review_order_before_submit';
});
```

**Comment by ouun** [comment]
Closing this as I have not heard back anything. Please feel free to reopen, if you have somne feedback. THX
